1.7/administration/high-availability/multi-region.md

```diff
@@ -0,0 +1,33 @@
+---
+post_title: Multiple Regions
+menu_order: 2
+---
+
+DC/OS supports multiple region configurations. This topic describes the setup recommendations and caveats.
+
+# Single Region Masters and Cross-Region Agents 
+DC/OS masters are within a region, possibly spanning zones. DC/OS agents span multiple regions. This setup is similar to masters in an on-prem datacenter and agents running on-prem and on public cloud. 
+
+#### Recommendations
+
+- This setup is typically not recommended because of how difficult it is to guarantee the latency and addressing requirements.
+- Masters within the region should be placed in different zones to tolerate zone failures.
+- DC/OS service schedulers should be constrained to the region that contains the masters.
+- The total cross-sectional bandwidth between regions will be extremely limited. We do not recommend deploying single data (I/O) intensive DC/OS services across multiple regions.
+
+#### Caveats
+
+- If the region that contains masters fails, apps in other regions will continue to run. However, app updates or rescheduling of failed apps is not possible until a quorum of masters come up.
+
+
+# Cross-Region Masters and Cross-Region Agents 
```

Comment:
Talked to @joel-hamill. We can list it for reference but we need to make it extra clear that this is not supported.